packages/cozy-client/src/store/index.js

```diff
@@ -82,7 +83,7 @@ export default combinedReducer
 export const createStore = () =>
   createReduxStore(
     combineReducers({ cozy: combinedReducer }),
-    applyMiddleware(thunk)
+    composeWithDevTools(applyMiddleware(thunk))
```

Comment:
I think we should only add it if the cozy is in debug. 

if(flag('debug')) ...


wdty?

Reply:
apparemment c'est pas la peine de restreindre en dev https://github.com/reduxjs/redux-devtools/blob/main/extension/docs/FAQ.md#how-to-disable-it-in-production

Et avec la nouvelle approche `configureStore` (utiliser `createStore` comme on fait est maintenant déprécié) ça active les devtools par défaut https://redux-toolkit.js.org/api/configureStore#devtools

Je suppose qu'on peut partir comme ça.

Reply:
par contre je comprends pas pourquoi dans l'app j'ai un `Can't resolve '@redux-devtools/extension' in 'node_modules/cozy-client/dist/store'` ?

Reply:
Je préfère éviter que n'importe qui puisse voir facilement son store redux. Je préfère qu'on laisse que si le flag est activé. Ça correspond pas à prod / dev chez eux. Au final, sur CC on affiche les devtools que si debug activé donc gardons la même logique ça me semble plus pertinent

Reply:
Je comprends que si, il parle bien d'un site en prod. J'ai mise à jour le lien qui pointe directement vers l'article en question.

Mais ok pour mettre la condition avec le flag pas de souci :)